### PR TITLE
Validate all translatable objects for choosing unexisiting locales in the database

### PR DESCRIPTION
**Pull request by TheMilek**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13                   |
| Bug fix?        | a bit                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| License         | MIT                                                          |

### CODE REVIEW

File: src/Sylius/Behat/Context/Api/Admin/TranslationContext.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public function __construct(
+        private ApiClientInterface $client,
+        private ResponseCheckerInterface $responseChecker,
+    ) {
+    }
+
+    /**
+     * @Then I should be notified that the locale is not available
+     */
+    public function IShouldBeNotifiedThatLocaleIsNotAvailable(): void
```
Comment (Wojdylak) [inline src/Sylius/Behat/Context/Api/Admin/TranslationContext.php]:
```suggestion
    public function iShouldBeNotifiedThatLocaleIsNotAvailable(): void
```

File: src/Sylius/Behat/Context/Api/Admin/TranslationContext.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Webmozart\Assert\Assert;
+
+final class TranslationContext implements Context
+{
+    public function __construct(
+        private ApiClientInterface $client,
```
Comment (Wojdylak) [inline src/Sylius/Behat/Context/Api/Admin/TranslationContext.php]:
```suggestion
        private readonly ApiClientInterface $client,
```

File: src/Sylius/Behat/Resources/config/services/contexts/api/admin.xml [hunk 3]
```diff
@@ -34,6 +34,11 @@
             <argument type="service" id="sylius.behat.shared_storage" />
         </service>
 
+        <service id="Sylius\Behat\Context\Api\Admin\TranslationContext" >
```
Comment (Wojdylak) [inline src/Sylius/Behat/Resources/config/services/contexts/api/admin.xml]:
```suggestion
        <service id="sylius.behat.context.api.admin.translation" class="Sylius\Behat\Context\Api\Admin\TranslationContext" >
```

**Review by Wojdylak — DISMISSED** [review]

**Review by jakubtobiasz — DISMISSED** [review]
👍🏻, but apply @Wojdylak's suggestions

**Review by Wojdylak — APPROVED** [review]

**Comment by Wojdylak** [comment]
Thank you, @TheMilek!

**Pull request merged**